modules/virt-create-node-network-config-console.adoc

```diff
@@ -0,0 +1,64 @@
+:_content-type: PROCEDURE
+[id="virt-create-node-network-config-console_{context}"]
+= Creating an interface on nodes
+
+You can create a policy by using either a form or YAML in the web console.
+
+.Procedure
+. Navigate to *Networking* → *NodeNetworkConfigurationPolicy*.
+
+. In the *NodeNetworkConfigurationPolicy* page, click *Create*, and select *From Form* option.
++
+In case there are no existing policies, you can alternatively click *Create NodeNetworkConfigurationPolicy* to create policy using form.
++
+[NOTE]
+====
+To create policy using YAML, click *Create*, and select *With YAML* option.
+====
+
+. Optional: Check the *Apply this NodeNetworkConfigurationPolicy only to specific subsets of nodes using the node selector* checkbox to specify the nodes where the policy must be applied.
+
+. Enter the policy name in *Policy name* field.
+
+. Optional: Enter the description of the policy in *Description* field.
+
+. Optional: In the *Policy Interface(s)* section, by default, a bridge interface is added. Update the list of network interfaces by executing the following steps:
```

Comment:
It should not be optional

Reply:
As per my observation in creating NNCP using form, I could see that I can create a policy without an interface. If that's not how it is supposed to be, then probably we might need a UI bug.
Can you confirm the desired behaviour?
cc: @upalatucci

Reply:
As of today, the user is able to create a policy with no interfaces. I'm not sure if this is a bug or not

Reply:
@upalatucci should we open a bug for discussion?